platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/test/java/com/swirlds/demo/consistency/ConsistencyTestingToolStateTest.java

```diff
@@ -0,0 +1,287 @@
+    private Round round;
+    private ConsensusEvent event;
+    private List<ScopedSystemTransaction<StateSignatureTransaction>> consumedTransactions;
+    private Consumer<ScopedSystemTransaction<StateSignatureTransaction>> consumer;
+    private Transaction consensusTransaction;
+    private StateSignatureTransaction stateSignatureTransaction;
+    private InitTrigger initTrigger;
+    private SoftwareVersion softwareVersion;
+    private Configuration configuration;
+    private ConsistencyTestingToolConfig consistencyTestingToolConfig;
+    private StateCommonConfig stateCommonConfig;
+
+    @BeforeAll
+    static void initState() {
+        state = new ConsistencyTestingToolState(FAKE_MERKLE_STATE_LIFECYCLES, mock(Function.class));
+        FAKE_MERKLE_STATE_LIFECYCLES.initStates(state);
+    }
+
+    @BeforeEach
+    void setUp() {
+        platform = mock(Platform.class);
+        initTrigger = InitTrigger.GENESIS;
+        softwareVersion = new BasicSoftwareVersion(1);
+        platformContext = mock(PlatformContext.class);
+        configuration = mock(Configuration.class);
+        consistencyTestingToolConfig = mock(ConsistencyTestingToolConfig.class);
+        stateCommonConfig = mock(StateCommonConfig.class);
+
+        when(platform.getSelfId()).thenReturn(NodeId.of(1L));
+        when(platform.getContext()).thenReturn(platformContext);
+        when(platformContext.getConfiguration()).thenReturn(configuration);
+        when(configuration.getConfigData(ConsistencyTestingToolConfig.class)).thenReturn(consistencyTestingToolConfig);
+        when(configuration.getConfigData(StateCommonConfig.class)).thenReturn(stateCommonConfig);
+        when(consistencyTestingToolConfig.freezeAfterGenesis()).thenReturn(Duration.ZERO);
+        when(stateCommonConfig.savedStateDirectory()).thenReturn(Path.of("consistency-test"));
+        when(consistencyTestingToolConfig.logfileDirectory()).thenReturn("consistency-test");
+
+        state.init(platform, initTrigger, softwareVersion);
+
+        main = mock(ConsistencyTestingToolMain.class);
+        random = new Random();
+        platformStateModifier = mock(PlatformStateModifier.class);
+        round = mock(Round.class);
+        event = mock(ConsensusEvent.class);
+
+        consumedTransactions = new ArrayList<>();
+        consumer = systemTransaction -> consumedTransactions.add(systemTransaction);
+        consensusTransaction = mock(TransactionWrapper.class);
+
+        final byte[] signature = new byte[384];
+        random.nextBytes(signature);
+        final byte[] hash = new byte[48];
+        random.nextBytes(hash);
+        stateSignatureTransaction = StateSignatureTransaction.newBuilder()
+                .signature(Bytes.wrap(signature))
+                .hash(Bytes.wrap(hash))
+                .round(round.getRoundNum())
+                .build();
+    }
+
+    @Test
+    void handleConsensusRoundWithApplicationTransactionEmptySystemTransactionList() {
```

Comment:
I think this method name can be simplified to just `handleConsensusRoundWithApplicationTransaction`